openldap-mixin/panels.libsonnet

```diff
@@ -0,0 +1,255 @@
+                          t.coreOperations('Delete', '__interval'),
+                        ],
+                        description='The key LDAP operations.'
+                      )
+                      + timeSeriesPanel.options.legend.withDisplayMode('table')
+                      + timeSeriesPanel.options.legend.withCalcsMixin(['min', 'max', 'mean'])
+                      + timeSeriesPanel.options.legend.withPlacement('right')
+                      + timeSeriesPanel.options.legend.withShowLegend(true)
+                      + timeSeriesPanel.standardOptions.withUnit('none')
+                      + timeSeriesPanel.standardOptions.withMin(0)
+                      + timeSeriesPanel.standardOptions.withDecimals(0)
+                      + timeSeriesPanel.standardOptions.color.withMode('palette-classic')
+                      + timeSeriesPanel.standardOptions.thresholds.withMode('absolute')
+                      + timeSeriesPanel.standardOptions.thresholds.withSteps([
+                        timeSeriesPanel.thresholdStep.withColor('green'),
+                      ]),
+
+      // Auxiliary Operations Panel
+      auxiliaryOperations: commonlib.panels.generic.timeSeries.base.new(
+                             'Auxiliary operations / $__interval',
+                             targets=[
+                               t.auxiliaryOperations('Abandon', '__interval'),
+                               t.auxiliaryOperations('Compare', '__interval'),
+                               t.auxiliaryOperations('Unbind', '__interval'),
+                               t.auxiliaryOperations('Extended', '__interval'),
+                               t.auxiliaryOperations('Modrdn', '__interval'),
+                             ],
+                             description='The less frequent but important LDAP operations.'
+                           )
+                           + timeSeriesPanel.options.legend.withDisplayMode('table')
+                           + timeSeriesPanel.options.legend.withCalcsMixin(['min', 'max', 'mean'])
+                           + timeSeriesPanel.options.legend.withPlacement('right')
+                           + timeSeriesPanel.options.legend.withShowLegend(true)
+                           + timeSeriesPanel.standardOptions.withUnit('none')
+                           + timeSeriesPanel.standardOptions.withMin(0)
+                           + timeSeriesPanel.standardOptions.withDecimals(0)
+                           + timeSeriesPanel.standardOptions.color.withMode('palette-classic')
+                           + timeSeriesPanel.standardOptions.thresholds.withMode('absolute')
+                           + timeSeriesPanel.standardOptions.thresholds.withSteps([
+                             timeSeriesPanel.thresholdStep.withColor('green'),
+                           ]),
+
+      // Primary Thread Activity Panel
+      primaryThreadActivity: commonlib.panels.generic.timeSeries.base.new(
```

Comment:
This needs to have its legend in table format
<img width="860" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/38274348/fe3e12f3-88c2-42a7-a59f-ca5b45d59cd4">